connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -299,7 +299,7 @@ public StatusBackingStore statusBackingStore() {
     }
 
     @Override
-    public ConfigInfos validateConnectorConfig(Map<String, String> connectorProps) {
+    public ConfigInfos validateConnectorConfig(Map<String, String> connectorProps, boolean doLog) {
```

Comment:
Related to the other comment, we'll need to keep the `@Override` of the previous method, and just have it call the new method that will also be overridden.